### PR TITLE
Video thumbnails

### PR DESCRIPTION
**Pull request by ignapas**
# Description

Uses thumbnail SciCrunch information for videos in the gallery

## Type of change

Delete those that don't apply.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually, but it is impossible to test properly until a dataset owner updates a dataset to include a thumbnail

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/ImagesGallery/ImagesGallery.vue [hunk 1]
```diff
@@ -364,22 +364,37 @@ export default {
...
+              thumbnailPaths[videoPath] = thumbnail.dataset.path
+            })
+          }
+          scicrunchData.video.forEach(async videoFile => {
+            let thumbnail = this.defaultVideoImg
+            if (thumbnailPaths[videoFile.dataset.path]) {
+              const url = new URL(`${process.env.portal_api}/s3-resource/${datasetId}/${datasetVersion}/files/${thumbnailPaths[videoFile.dataset.path]}`)
+              url.searchParams.append('encodeBase64', true)
+              const img = await fetch(url).then(resp => resp.ok ? resp.text() : null)
+              console.log(url.toString(), img)
```
Comment (egauzens) [inline components/ImagesGallery/ImagesGallery.vue:380]: please remove the log statement

**Review by egauzens — COMMENTED** [review]
How did you test locally? Were you able to update a dataset on the curation index or something?

File: components/ImagesGallery/ImagesGallery.vue (hunk 1 above)
Comment (ignapas) [reply components/ImagesGallery/ImagesGallery.vue]: done

**Comment by ignapas** [comment]
> How did you test locally? Were you able to update a dataset on the curation index or something?

I was only able to hack some stuff through payload data injection hardcoding the ID of a test, unpublished dataset for the SciCrunch calls. I think the only way to test and be sure it works is modifying a publish dataset that contains a video, adding a thumbnail and editing the manifest file appropriately.

**Review by egauzens — APPROVED** [review]

**Pull request merged**